### PR TITLE
kubectl interactive plugin manifest added to krew-index

### PR DESCRIPTION
**Pull request by emreodabas**
-----
interactive plugin -> 
https://github.com/emreodabas/kubectl-plugins/blob/master/README.md#kubectl-interactive-in-krew-repo-same-as-kubectl-it
**Checklist for plugin developers:**

- [x] Read the [Plugin Naming Guide](https://github.com/GoogleContainerTools/krew/tree/master/docs/NAMING_GUIDE.md) (for new plugins)
- [x] Verify the installation from URL or a local archive works (`kubectl krew install --manifest=[...] --archive=[...]`)

### CODE REVIEW

**Comment by ahmetb** [comment]
Do you mind doing `git commit --amend --no-edit && git push -f` sometime, so that the newly added testing system tests this manifest?

File: plugins/interactive.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: interactive               
+spec:
+  version: "v1.1.0-beta"      
+  platforms:
+  - selector:             
+      matchLabels:
+        os: linux
```
[the remark this answers is not in the document]
Comment (emreodabas) [reply plugins/interactive.yaml]: I actually not tested in macOS before, today I checked and it seems to work in macOS. I have added darwin to manifest

File: plugins/interactive.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    uri: https://github.com/emreodabas/kubectl-plugins/archive/v1.1.0-beta.tar.gz
+    sha256: "80f7b55d82673253eb0957d2c248163fcce3e4c3bc609410b1e9ee2d6d6a91a7"
+    files:                     
+    - from: kubectl-plugins-1.1.0-beta/kubectl-it
```
[the remark this answers is not in the document]
Comment (emreodabas) [reply plugins/interactive.yaml]: thanks, I did it.

File: plugins/interactive.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: interactive
```
Comment (juanvallejo) [inline plugins/interactive.yaml]: I'd recommend a more specific name, [per the naming guide](https://github.com/GoogleContainerTools/krew/tree/master/docs/NAMING_GUIDE.md). Perhaps something like `interactive-prompt`, `prompt-shell`, `interactive-shell`, or `it-shell`

**Comment by ahmetb** [comment]
Do you mind explaining how does your plugin differ from https://github.com/cloudnativelabs/kube-shell ? The word “interactive” is too common and can be a name grab, given there are several projects that do the same job.

**Comment by emreodabas** [comment]
I did not know that kube-shell, it seems way better than my plugin. From this point plugin is useless. We could close this PR.

**Comment by ahmetb** [comment]
Thanks!

**Pull request closed without merging**